dashboard/src/components/BindingList/BindingListEntry.tsx

```diff
@@ -10,57 +11,66 @@ interface IBindingListEntryProps {
 }
 
 interface IBindingListEntryState {
-  isExpanded: boolean;
+  modalIsOpen: boolean;
 }
 
 class BindingListEntry extends React.Component<IBindingListEntryProps, IBindingListEntryState> {
   public state = {
-    isExpanded: false,
+    modalIsOpen: false,
   };
 
   public render() {
     const {
       bindingWithSecret,
       bindingWithSecret: { binding },
     } = this.props;
-    const { name, namespace } = binding.metadata;
+    const { name } = binding.metadata;
 
-    const condition = [...binding.status.conditions].shift();
-    const currentStatus = condition ? (
-      <div className="condition">
-        <code>{condition.message}</code>
-      </div>
-    ) : (
-      undefined
-    );
+    let reason = <span />;
+    let message = "";
+    const condition = [...binding.status.conditions]
+      .sort((a, b) => a.lastTransitionTime.localeCompare(b.lastTransitionTime))
+      .pop();
+    if (condition) {
+      reason = <code>{condition.reason}</code>;
+      message = condition.message;
+    }
 
-    const rows = [
-      <tr key={"row"}>
-        <td>
-          {namespace}/{name}
-        </td>
-        <td>{currentStatus}</td>
-        <td style={{ display: "flex", justifyContent: "space-around" }}>
-          <button className={"button button-primary button-small"} onClick={this.toggleExpand}>
-            Expand/Collapse
+    return (
+      <tr className="row padding-t-small">
+        <td className="col-2">{name}</td>
+        <td className="col-2">{reason}</td>
+        <td className="col-2">
+          <button className="button" onClick={this.openModal}>
```

Comment:
I think a small button will work better :)

```suggestion
          <button className="button button-small" onClick={this.openModal}>
```